fml/trace_event.h

```diff
@@ -45,8 +46,9 @@
 // from trace/event.h on Fuchsia.
 //
 // TODO(chinmaygarde): All macros here should have the FML prefix.
-#define FML_TRACE_COUNTER(category_group, name, count) \
-  ::fml::tracing::TraceCounter(category_group, name, count);
+#define FML_TRACE_COUNTER(category_group, name, counter_id, arg1, ...)         \
+  ::fml::tracing::TraceCounter((category_group), (name), (counter_id), (arg1), \
```

Comment:
This isn't actually what I meant, but if it works, great.

Reply:
Seemed to work on Windows. What did you actually mean?

Reply:
I meant that the way simulate variadic macros on platforms that don't support them is to do:

    #define MACRO(args) (some_variadic_function args)

and then invoke it as:

    MACRO((arg1, arg2, arg3, arg4));

Reply:
How is that ... variadic?

Reply:
You can invoke the macro with any number of "arguments" and have them forward to an actual variadic function.  It'd behave like a variadic macro except that you'd have to live with extra parentheses wherever it's used.